.buildkite/pipeline.yml

```
@@ -20,7 +20,6 @@ steps:
     matrix:
       setup:
         version:
-          - "1.6"
           - "1"
```

```suggestion
          - "1.9"
          - "1"
```
If we set the lower version to v1.9, shouldn't we run tests specifically on Julia v1.9 (given that v1 may soon be v1.10)?